.ci/generate-releases.sh

```diff
@@ -22,6 +22,9 @@
 #
 set -eo pipefail
 
+if [ -n "$RUNNER_DEBUG" ] ; then
```

Comment:
help debugging when running in a debug mode through the [GitHub actions](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging)